meta-mender-core/classes/mender-state-scripts.bbclass

```diff
@@ -14,6 +18,8 @@ do_create_default_script_dirs() {
     mkdir -p ${S}/mender-state-scripts ${B}/mender-state-scripts
 }
 addtask do_create_default_script_dirs after do_patch before do_configure
+# Clean stale state scripts from previous builds to ensure reproducibility
+do_create_default_script_dirs[cleandirs] = "${S}/mender-state-scripts ${MENDER_STATE_SCRIPTS_DIR}"
```

Comment:
I see that we just added the explicit dirs that we create in the task `do_create_default_script_dirs`. However, should we in both cases use the variable `MENDER_STATE_SCRIPTS` defined above?

```suggestion
do_create_default_script_dirs[cleandirs] = "${MENDER_STATE_SCRIPTS}"
```

Reply:
@lluiscampos after looking more closely, I actually think we can get rid of the task altogether. Can you give the current state a look?